src/features/pages/home/live-pricing/table-component/live-market-table.tsx

```diff
@@ -0,0 +1,158 @@
+
+export type TLiveMarketTableProps = {
+    selected_market: TAvailableLiveMarkets
+    link_to: string
+    display_name: string
+}
+
+const LiveMarketTable = ({ selected_market, link_to, display_name }: TLiveMarketTableProps) => {
+    const { is_eu } = useRegion()
+    const [initial_loaded, setInitialLoaded] = useState(false)
+
+    const [rawMarketsData, setRawMarketsData] = useState()
+
+    const TABLE_VISIBLE_ROWS = 5
+    const [markets_data, setMarketsData] = useState(() => {
+        const temp = new Map<TAvailableLiveMarkets, TMarketData[]>()
+        return temp
+    })
+    const intervalRef = useRef(null)
+
+    const [sorting, setSorting] = React.useState<SortingState>([])
+
+    const region = useMemo(() => {
+        return is_eu
+            ? 'https://deriv-static-pricingfeed.firebaseio.com/eu.json'
+            : 'https://deriv-static-pricingfeed.firebaseio.com/row.json'
+    }, [is_eu])
+
+    const getData = useCallback(async () => {
+        const rawResponse = await fetch(region)
+        const response = await rawResponse.json()
+        setRawMarketsData(response)
+    }, [region])
+
+    useEffect(() => {
+        if (!initial_loaded) {
+            getData()
+            setInitialLoaded(true)
+        }
+        intervalRef.current = setInterval(getData, 10000)
+        return () => {
+            clearInterval(intervalRef.current)
+        }
+    }, [getData, initial_loaded])
+
+    const updateData = useCallback(() => {
+        if (rawMarketsData) {
+            const stocks = rawMarketsData.market['stocks']
+            const indices = rawMarketsData.market['indices']
+            const stocks_indices = Object.assign(stocks, indices)
+            const res = { ...rawMarketsData.market, indices: { ...stocks_indices } }
+
+            Object.keys(res).map((item) => {
+                if (item == selected_market) {
+                    const selected_market_data = res[item]
+                    const result = Object.values(selected_market_data)
+                    setMarketsData(result)
+                }
+            })
+        }
+    }, [rawMarketsData, selected_market])
+
+    useEffect(() => {
+        updateData()
+    }, [updateData])
+
+    const columns = useLiveColumns()
+
+    const table = useReactTable({
+        data: markets_data,
+        columns,
+        state: {
+            sorting,
+        },
+        getCoreRowModel: getCoreRowModel(),
+        getSortedRowModel: getSortedRowModel(),
+        onSortingChange: setSorting,
+    })
+
+    const rows = table.getRowModel().rows.slice(0, TABLE_VISIBLE_ROWS)
+
+    return (
+        <>
+            <Flex.Box justify="center" mt="16x">
+                <table>
+                    <thead>
+                        {table.getHeaderGroups().map((headerGroup) => (
+                            <Flex.Box key={headerGroup.id} justify="center">
+                                <tr className={table_row_header}>
```

Comment:
I remember that already advice this approach, it is work?
If yes, we can pass most of properties from css file here

And instead of set `inline-size` for each device, let's play with `flex-basis`
If you need help with that, please let me know 🙏 
```suggestion
                                <Box as="tr" className={table_row_header}>
```